Source/WebGPU/WebGPU/Texture.mm

```diff
@@ -2417,17 +2432,26 @@ static WGPUExtent3D computeRenderExtent(const WGPUExtent3D& baseSize, uint32_t m
 
     auto slices = NSMakeRange(descriptor->baseArrayLayer, descriptor->arrayLayerCount);
 
-    id<MTLTexture> texture = [m_texture newTextureViewWithPixelFormat:pixelFormat textureType:textureType levels:levels slices:slices];
-    if (!texture)
-        return TextureView::createInvalid(m_device);
+    id<MTLTexture> texture;
```

Comment:
What happens when you bind the texture to a bind group? I think we'd need to have affordances for that (probably by just unconditionally giving it backing storage immediately?)